app/packages/partup-server/methods/sectors/sectors_methods.js

```
@@ -40,11 +40,32 @@ Meteor.methods({
         if (!User(user).isAdmin()) throw new Meteor.Error(401, 'unauthorized');
 
         try {
-            Networks.update({sector: sectorId}, {$unset: {'sector': ''}});
+            Networks.update({sector_id: sectorId}, {$unset: {'sector_id': ''}});
             Sectors.remove({_id: sectorId});
         } catch (error) {
             Log.error(error);
             throw new Meteor.Error(400, 'sector_could_not_be_removed');
         }
-    }
+    },
+
+    /**
+     * Update an existing sector
+     * @name sectors.update
+     * @param {String} sectorId
+     * @param {Mixed[]} fields
+     */
+     'sectors.update': function (sectorId, fields) {
+         check(sectorId, String)
+         check(fields, Partup.schemas.forms.sector)
+
+         if (!User(Meteor.user()).isAdmin()) throw new Meteor.Error(401, 'unauthorized')
+
+         let sector = Sectors.findOneOrFail({_id:sectorId})
```

@wouterd I'm still not sure where you want to throw the 404 error. The '.findOneOrFail' already throws a 404 error if it can't find the item it's looking for. It might be this behavior is not like in C# as I'm used to and it still continues the method call even with the error?